vercel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "crons": [
+    {
+      "path": "/api/storeNlpData",
+      "schedule": "0 0 * * *"
+    }
+  ]
+}
```

Comment:
how to configure a cron job in vercel. video for reference: http://xn--%22%22-kf0c/